### ISSUE TITLE
ERROR: /apollo/bazel-apollo/modules/prediction/predictor/free_move/BUILD:44:1: in py_test rule //bazel-apollo/modules/prediction/predictor/free_move:free_move_predictor_test_cpplint: paths to Python

### ISSUE THREAD

**Issue by zhaochongjie** [ApolloAuto/apollo#5990]
INFO: Reading 'startup' options from /apollo/tools/bazel.rc: --batch_cpu_scheduling
ERROR: /apollo/bazel-apollo/modules/prediction/predictor/free_move/BUILD:44:1: in py_test rule //bazel-apollo/modules/prediction/predictor/free_move:free_move_predictor_test_cpplint: paths to Python packages may not contain '-'.
ERROR: Analysis of target '//bazel-apollo/modules/prediction/predictor/free_move:free_move_predictor_test_cpplint' failed; build aborted.
INFO: Elapsed time: 34.072s
INFO: Reading 'startup' options from /apollo/tools/bazel.rc: --batch_cpu_scheduling
ERROR: /apollo/bazel-apollo/modules/perception/onboard/BUILD:63:1: in py_test rule //bazel-apollo/modules/perception/onboard:subnode_test_cpplint: paths to Python packages may not contain '-'.
ERROR: Analysis of target '//bazel-apollo/modules/perception/onboard:subnode_test_cpplint' failed; build aborted.
INFO: Elapsed time: 1.471s

**Comment by Wang4758** [comment]
Hi, did you manage to solve this error?

**Comment by zhangxingjing** [comment]
@Wang4758,@zhaochongjie,HI,have you solved this problem?

**Comment by storypku** [comment]
> @Wang4758,@zhaochongjie,HI,have you solved this problem?

It seems that this bug was already fixed in Apollo `master`, after 1.5 yrs this issue was put up.

```
$ bazel test --config=cpplint modules/prediction/predictor/free_move/...
Starting local Bazel server and connecting to it...
(12:34:08) INFO: Current date is 2020-07-17
(12:34:11) INFO: Analyzed 2 targets (28 packages loaded, 432 targets configured).
(12:34:11) INFO: Found 2 test targets...
(12:34:11) INFO: Elapsed time: 8.308s, Critical Path: 0.35s
(12:34:11) INFO: 4 processes: 4 local.
(12:34:11) INFO: Build completed successfully, 7 total actions
//modules/prediction/predictor/free_move:free_move_predictor_cpplint     PASSED in 0.3s
//modules/prediction/predictor/free_move:free_move_predictor_test_cpplint PASSED in 0.3s

(12:34:11) INFO: Build completed successfully, 7 total actions
```

**Comment by storypku** [comment]
> > @Wang4758,@zhaochongjie,HI,have you solved this problem?
> 
> It seems that this bug was already fixed in Apollo `master`, after 1.5 yrs this issue was put up.
> 
> ```
> $ bazel test --config=cpplint modules/prediction/predictor/free_move/...
> Starting local Bazel server and connecting to it...
> (12:34:08) INFO: Current date is 2020-07-17
> (12:34:11) INFO: Analyzed 2 targets (28 packages loaded, 432 targets configured).
> (12:34:11) INFO: Found 2 test targets...
> (12:34:11) INFO: Elapsed time: 8.308s, Critical Path: 0.35s
> (12:34:11) INFO: 4 processes: 4 local.
> (12:34:11) INFO: Build completed successfully, 7 total actions
> //modules/prediction/predictor/free_move:free_move_predictor_cpplint     PASSED in 0.3s
> //modules/prediction/predictor/free_move:free_move_predictor_test_cpplint PASSED in 0.3s
> 
> (12:34:11) INFO: Build completed successfully, 7 total actions
> ```
